ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -358,6 +360,10 @@ public boolean isLeader() {
     return role.isLeader();
   }
 
+  public boolean isActive() {
+    return !lifeCycle.getCurrentState().isPausingOrPaused();
+  }
```

Comment:
When the state is CLOSED, isActive() returns true.  It does not sound right.  Let's rename this method to isPausingOrPaused?

Reply:
SG. Updated.